src/Ylmish/Adaptive.Codec.fs

```diff
@@ -279,6 +279,11 @@ module Decoder =
     let inline tryParse (path : Path, element : 'a) : Decoded<'b> =
         let mutable value = Unchecked.defaultof< ^b>
         let result = (^b: (static member TryParse: 'a * byref< ^b> -> bool) element, &value)
+        printfn $"a (input) is {typeof<'a>}"
+        printfn $"b (output) is {typeof<'b>}"
+        printfn $"input is {element}"
+        printfn $"output is {value}"
+        printfn $"parsed {result}"

```

Comment:
This logs:
```
[Fable/Mocha] a (input) is System.String
[Fable/Mocha] b (output) is System.Int32
[Fable/Mocha] input is 42
[Fable/Mocha] output is 0
[Fable/Mocha] parsed false
```

🙃

Reply:
Replacing the SRTP parsing with an explicit Int32 parser fixed things.

I imagine the doubly inlined tryParse functions (Decode.tryParse, Decoder.tryParse) might have meant the fable compiler's [special cases](https://github.com/fable-compiler/Fable/blob/8c0767d4426c697e04421194ab1d265bfd8e597e/src/Fable.Transforms/FSharp2Fable.Util.fs#L881-L924) weren't matching. 🤷‍♂️